src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.first-search {
+  position: relative;
+  display: block;
+}
+
+.second-search {
+  position: relative;
+  display: block;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
Consider using variables for repeating colors

Reply:
Move all repeatable colors to the variables